sections/loans/components/ActionBox/ActionBox.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useRouter } from 'next/router';
+import StructuredTab from 'components/StructuredTab';
+import { ACTION_BOX_WIDTH, LOAN_TYPE_ETH } from 'sections/loans/constants';
+import BorrowSynthsTab from './BorrowSynthsTab/BorrowSynthsTab';
+import ActiveBorrowsTab from './ActiveBorrowsTab/ActiveBorrowsTab';
+
+type ActionBoxProps = {};
+
+const ActionBox: React.FC<ActionBoxProps> = ({}) => {
+	const { t } = useTranslation();
+	const router = useRouter();
+
+	const action = router.query.action!;
+	let currentTab = 'list';
+	let loanType: string = '';
+	let loanId: string = '';
+	let loanAction: string = '';
+
+	if (action) {
+		loanType = action[0];
+	}
+	if (!loanType || loanType === 'new') {
+		currentTab = 'new';
+	} else {
+		loanId = action[1];
+		loanAction = action[2];
+	}
```

Comment:
for the code above ^ you could use `useMemo` here no?

Reply:
Sure.